packages/connect-common/files/coins.json

```diff
@@ -2236,7 +2236,7 @@
             "support": {
                 "T1B1": false,
                 "T2B1": "2.6.1",
-                "T2T1": "2.0.8",
+                "T2T1": "2.4.3",
```

Comment:
maybe bumping it here https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/data/config.ts#L124 
would make more sense. 
coins.json is more like "firmware support". 
config.ts on the other hand can be seen as "connect" support

Reply:
and in config.ts, I would go similary to "bnb" record
```
{
            coin: ['bnb'],
            min: { T1B1: '1.9.0', T2T1: '2.3.0' },
            comment: [
                'There were protobuf backwards incompatible changes with introduction of 1.9.0/2.3.0 firmwares',
            ],
        },
```
just all ada, and tada coins are not allowed for version lower than 2.4.3

Reply:
Done.